design/work_item_link_type.go

```diff
@@ -76,26 +76,9 @@ var workItemLinkTypeRelationships = a.Type("WorkItemLinkTypeRelationships", func
 	a.Description(`JSONAPI store for the data of a work item link type.
 See also http://jsonapi.org/format/#document-resource-object-relationships`)
 	a.Attribute("link_category", relationWorkItemLinkCategory, "The work item link category of this work item link type.")
```

Comment:
To keep it here: why wilt doesn't it have any reference ID to the combination table ?